src/SmartTransactionsController.test.ts

```diff
@@ -337,7 +337,8 @@ describe('SmartTransactionsController', () => {
     nock.cleanAll();
   });
 
-  it('initializes with default state', async () => {
+  // update test name to reflect the new persistence enabled state
+  it('initializes with default state and persistence enabled', async () => {
```

Comment:
There’s no need to update this test either, as handling persistence is not the responsibility of this controller, and we typically don’t verify persistence in our controllers.

Reply:
I see your point. I will make some adjustments. I appreciate the feedback.